contactmps/views.py

```diff
@@ -46,16 +34,15 @@ def embed(request):
 def create_mail(request):
     # Only retuns persons with at least one email address
     # Count the number of emails we've sent them
-    neglected_persons = Person.objects \
-                              .filter(contactdetails__type='email') \
-                              .annotate(num_email_addresses=Count('contactdetails')) \
-                              .annotate(num_emails=Count('email')) \
-                              .prefetch_related('party', 'contactdetails') \
-                              .order_by('num_emails')[:4]
     persons = Person.objects \
-        .prefetch_related('party', 'contactdetails') \
-        .all()
+        .filter(contactdetails__type='email') \
+        .annotate(num_emails=Count('email')) \
+        .prefetch_related('party', 'contactdetails')
+
+    # of those MPs that are less emailed, randomly choose 4
+    neglected_persons = sorted(persons, key=lambda p: (p.num_emails, random.random()))[:4]
```

Comment:
@jbothma pls sanity check my logic here? I want to put the least emailed first, then randomly choose 4.